components/object-property-list/object-property-list-item-link.js

```diff
@@ -29,29 +29,21 @@ class ObjectPropertyListItemLink extends FocusMixin(ObjectPropertyListItem) {
 		};
 	}
 
-	static get styles() {
-		return [
-			...super.styles,
-			linkStyles
-		];
-	}
-
 	static get focusElementSelector() {
-		return '.d2l-link';
+		return 'd2l-link';
 	}
 
 	render() {
 		return html`
 			${this._renderIcon()}
 			${!this.skeleton ? html`
-				<a
+				<d2l-link
+					aria-label="${this.text}"
```

Comment:
The `aria-label` shouldn't be needed? I'd expect its label to come from its internals via `renderText()`.

Reply:
I'd mostly added it since it was a property of `d2l-link`, but you're right that it should just be handled within `renderText`. Though it does beg the question, if this component should have some property to provide more context to screen reader users like how `aria-label` is currently doing for `d2l-link`. I'm good with just removing it for now though!

Reply:
Setting `aria-label` in this way actually replaces the contents, so there wouldn't be more context in the same way `aria-description` might provide.

Having `aria-label` on `<d2l-link>` as a passthrough to its underlying `<a>`'s `aria-label` was a mistake we made initially. All the `aria-*` attribute are obviously native attributes and aXe has actually started complaining recently when we put them on custom elements.

What we should have done (and have correct in newer things) is to use a combination of `text` (or `label`) and `text-hidden`/`label-hidden` to flip that to the `aria-label` of the native element. And then add something like a `description` attribute to go into the `aria-description`.